Standards/CCSMB-14.md

```diff
@@ -0,0 +1,56 @@
+# *CCSMB 14:* Stereo television
+
+*Author: @notbronwyn*  
+*Version: v1.0*  
+*Last updated: 2024-03-30*  
+
+## Server:
+Videos should either be in BIMG but other formats do not break compliance as long as they are marked in "type" to avoid issues with clients and there is an open-source encoder and decoder
+  
+  an example of this would be [PBB](https://raw.githubusercontent.com/craftytv/pigutv/main/pbbconverter.py) which is supported in the example server
+
+Broadcasts in BIMG should only use the default CraftOS pallete
+  
+Each frame of video should have enough audio to last the entire frame and no more
```

Comment:
And how about it being shorter than the frame, is that compliant. Also, "should" shouldnt be used here

Reply:
> And how about it being shorter than the frame, is that compliant.

No, i forgot to mention that